<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Heart Anatomy Training Tool</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="vtk_layout">
      <item alignment="Qt::AlignLeft">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Segments</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="9" column="0">
          <widget class="QCheckBox" name="stereo_box">
           <property name="text">
            <string>ON/OFF</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QCheckBox" name="AA_box">
           <property name="text">
            <string>Ascending aorta</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="Learning_mode">
           <property name="text">
            <string>Clear segment colors</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="LAC_box">
           <property name="text">
            <string>Left atrial cavity</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="LVC_box">
           <property name="text">
            <string>Left ventricular cavity</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="RAC_box">
           <property name="text">
            <string>Right atrial cavity</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1" alignment="Qt::AlignHCenter">
          <widget class="QCheckBox" name="LVM_Vis">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QCheckBox" name="PA_box">
           <property name="text">
            <string>Pulmonary artery</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QCheckBox" name="RVC_box">
           <property name="text">
            <string>Right ventricular cavity</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="LVM_box">
           <property name="text">
            <string>Left ventricle myocardium</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1" alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Visible</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" alignment="Qt::AlignHCenter">
          <widget class="QCheckBox" name="LAC_Vis">
           <property name="text">
            <string/>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="6" column="1" alignment="Qt::AlignHCenter">
          <widget class="QCheckBox" name="AA_Vis">
           <property name="text">
            <string/>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="1" alignment="Qt::AlignHCenter">
          <widget class="QCheckBox" name="LVC_Vis">
           <property name="text">
            <string/>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="1" alignment="Qt::AlignHCenter">
          <widget class="QCheckBox" name="RAC_Vis">
           <property name="text">
            <string/>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="1" alignment="Qt::AlignHCenter">
          <widget class="QCheckBox" name="RVC_Vis">
           <property name="text">
            <string/>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="7" column="1" alignment="Qt::AlignHCenter">
          <widget class="QCheckBox" name="PA_Vis">
           <property name="text">
            <string/>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QFrame" name="heart_frame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
         <item row="8" column="0" alignment="Qt::AlignBottom">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Stereo Rendering</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="vtk_layout2">
      <item>
       <widget class="QFrame" name="heart_frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>839</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
